docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -0,0 +1,91 @@
+
+= Cortex Time Series Plugin
+
+The Time Series Storage Plugin for Cortex helps {page-component-title} to persist aggregated flow data to Cortex.
+The plugin uses the xref:deployment:time-series-storage/timeseries/ts-integration-layer.adoc[time series storage integration layer] to convert metrics to a Prometheus model and delegate writes and reads to https://cortexmetrics.io/[Cortex].
+The plugin also makes it possible for https://docs.opennms.com/helm/latest/index.html[OpenNMS Helm] to visualize flow graphs from the Cortex instance.
+
+.Time Series Storage Plugin for Cortex architecture
+image::time-series-storage/cortex.png[Cortex plugin architecture,500]
+
+== Requirements
+
+Make sure you have the following before you start to use the Cortex plugin:
+
+* Horizon 30.0.0+ (Meridian availability with the 2023 release)
+* Cortex version 1.10.0+
+
+NOTE: We plan to keep the plugin compatible with https://grafana.com/oss/mimir/[Mimir], Grafana's fork of Cortex, as long as Mimir remains compatible with Cortex.
+The minimum supported Mimir version is 2.0.0.
+
+== Deployment
+
+To use the Time Series Storage Plugin for Cortex, do the following:
+
+.Start Cortex
+See https://cortexmetrics.io/docs/getting-started/.
+
+You can also download a https://github.com/opennms-forge/stack-play/tree/master/standalone-cortex-minimal[minimal, standalone version of Cortex] and start Cortex with `docker-compose up`.
+
+.Build and install the plugin into your local Maven repository
```

Comment:
We should be able to eliminate this developer-centric step, replacing it with instructions for downloading the pre-built plugin from GitHub or another place. Details will become available closer to the September release day.

Reply:
Do we have a general idea of how this will work so that we can rough it in?